lib/views/desktop/send/widgets/DTSelectOrDropAFile.dart

```diff
@@ -21,15 +23,30 @@ class DTSelectOrDropAFile extends StatefulWidget {
 class _DTSelectOrDropAFile extends State<DTSelectOrDropAFile> {
   @override
   Widget build(BuildContext context) {
+    bool flag = false;
```

Comment:
What is `flag`? If it is to indicate that drag-drop is ongoing, can it be called something else like `isDragging` or similar?